Makefile

```diff
@@ -0,0 +1,16 @@
+develop:
+	python setup.py develop
+
+undevelop:
+	python setup.py develop --uninstall
+
+test:
+	flake8 rds-create-freestoragespace-alarms
```

Comment:
You wrote `rds-create-freestoragespace-alarms git:(import-all) flake8 rds_create_freestoragespace_alarms/**/*.py` and in circle.yml you wrote `flake8 rds-create-freestoragespace-alarms`.

`-` vs `_`.